birdhouse/components/stac/config/twitcher/docker-compose-extra.yml

```diff
@@ -0,0 +1,10 @@
+version: "3.4"
+
+services:
+  # extend twitcher with MagpieAdapter hooks employed for stac proxied requests
+  twitcher:
+    volumes:
+      # NOTE: MagpieAdapter hooks are defined within Magpie config, but it is actually Twitcher proxy that runs them
+      # target mount location depends on main docker-compose 'MAGPIE_PROVIDERS_CONFIG_PATH' environment variable
+      - ./components/stac/config/magpie/config.yml:/opt/birdhouse/src/magpie/config/stac-config.yml:ro
```

Comment:
I realise that this comment came from Weaver example, but it is somewhat confusing looking back at it. 
`MAGPIE_PROVIDERS_CONFIG_PATH` is used in other places with a different value (ie: defined in https://github.com/bird-house/birdhouse-deploy/blob/master/birdhouse/config/magpie/default.env#L28 and everywhere else that it is used). 

Can you adjust the comment (here and in Weaver's corresponding config) to have a reference to `birdhouse/config/twitcher/docker-compose-extra.yml` where this "other" `MAGPIE_PROVIDERS_CONFIG_PATH` mentioned here actually refers to?

Reply:
Yes I find this confusing too.  I think what Francis meant is to reference the location where the variable `MAGPIE_PROVIDERS_CONFIG_PATH` is actually defined, here https://github.com/bird-house/birdhouse-deploy/blob/547cc085c52924a21d853eeb1149790c2cb7577d/birdhouse/config/twitcher/docker-compose-extra.yml#L21

So we know which definition is actually "in effect".

Just curious, why do we have 2 flavors of the same variable?  Because one is the Magpie container and the other is Twitcher container?

Reply:
Indeed. This is what I meant.

The 2 flavors is because one is inside Magpie, and the other in Twitcher, but both are loaded by Magpie code (`MagpieAdapter` specifically in the case of Twticher). 
Inside Twitcher, the directory structure is not that important because other files are not used.
The only thing to respect is the multi-config loading strategy, as described in the comment just above that variable definition.